packages/sources/finage-test/src/endpoint/http/stock.ts

```diff
@@ -0,0 +1,37 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { EndpointTypes } from '../stock-router'
+
+export const httpTransport = new HttpTransport<EndpointTypes>({
+  prepareRequests: (params, config) => {
+    return {
+      params,
+      request: {
+        baseURL: config.API_ENDPOINT,
+        url: '/last/stocks',
+        params: {
+          symbols: [...new Set(params.map((p) => p.base.toUpperCase()))].join(','),
+          apikey: config.API_KEY,
+        },
+      },
+    }
+  },
+  parseResponse: (_, res) => {
+    // Filter null values in response
+    const response = res.data.filter((e) => e)
+
+    return response.map((entry) => {
+      return {
+        params: { base: entry.symbol },
+        response: {
+          data: {
+            result: entry.bid,
```

Comment:
Not entirely sure why v2 behaves this way but I think when using the batch endpoint the result is supposed to be (ask + bid) / 2. There is a TODO on the batch handler to "check this" so maybe this is something I'll have to check with Data Feeds on.

Reply:
+1 on this. I have sent a message to Data Feeds for confirmation

Reply:
Confirmed we should do (ask + bid) / 2